guide/private-key-management/schemes.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: guide
+title: Private key schemes
+nav_order: 1
+parent: Private key management
+permalink: /guide/private-key-management/schemes/
+main_classes: -no-top-padding
+image: /assets/images/guide/private-key-management/schemes/page-private-key-schemes.jpg
+---
+
+<!--
+
+Editor's notes
+
+An overview of what different private key management schemes there are, and how to choose one depending on use-case.
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/private-key-management/schemes.jpg"
+   retina = "/assets/images/guide/private-key-management/schemes@2x.jpg"
+   mobile = "/assets/images/guide/private-key-management/schemes-mobile.jpg"
+   mobileRetina = "/assets/images/guide/private-key-management/schemes-mobile@2x.jpg"
+   alt-text = "Schemes header illustration"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Private key schemes
+The private key controls the bitcoin [wallet]({{ '/guide/glossary/#wallet' | relative_url }}){:target="_blank"}, and the *scheme* is simply a way to describe the way that the key is stored and kept secure.
+
+Important aspects to consider when choosing a private key management scheme is what you expect your users to use the wallet-application for, who they are and how much bitcoin they might store in your product. For example:
+
+
+* **Target audience** - Are your users completely new to bitcoin and its concepts, or well versed in all the technological underpinnings?
+* **Use-case** - Are you building a product for daily payments, or a long-term life-saving storage solution?
+* **Value stored** - While we always strive for no loss of funds, how critical to their financial situation would it be if your customer lost access to their funds?
+
+
+The narrower you can define your answer to these questions, the easier it will be to pick the right private key management scheme, and provide a great user experience. If you find yourself wanting to target a wide range, this is a good signal to consider how you can employ a progressive scheme that grows with the customer. 
+
+## Balancing security, risk and convenience
+
+The trick to picking a private key management scheme is finding the appropriate balance between convenience on the one hand, and risk of loss and redundant security on the other. While continuous improvements mean we can now enable higher security and good user experience at the same time, there is always a balance. Higher security generally means more friction and work for the user, both during setup and usage of a wallet-application. 
```

Comment:
What do you mean by `continuous improvements`? Releasing new software updates regularly? Or that private key management schemes can change over time?

Reply:
Rephrased. It was a reference to learnings over the last decade.